pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.zero_x_baadf00d</groupId>
     <artifactId>partialize</artifactId>
-    <version>20.05</version>
+    <version>20.10</version>
```

Comment:
Do not change the version in this commit. It will be modified in the next release.